### PR TITLE
IAP-5429 - Update release notes August 

### PR DESCRIPTION
**Pull request by dginevaAxway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

1. Release notes was updated with new API Portal Catalog optimizations , updated related to August release and fixed issues list 
2. Update API Portal was updated to point latest, not May 2022 as was for previews release 

## Deploy preview link

https://deploy-preview-2404--axway-open-docs.netlify.app/docs/apim_relnotes/20220830_apip_relnotes/
https://deploy-preview-2404--axway-open-docs.netlify.app/docs/apim_installation/apiportal_install/upgrade_automatic/#upgrade-from-the-joomla-administrator-interface
https://deploy-preview-2404--axway-open-docs.netlify.app/docs/apim_installation/apiportal_docker/docker_portal_upgrade/

## Checklist for contributors

Before submitting this PR, please make sure:

_This is simply a reminder of what we are going to look for before merging your change._

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

### CODE REVIEW

File: content/en/docs/apim_relnotes/20220830_apip_relnotes.md [hunk 1]
```diff
@@ -33,40 +33,40 @@ We highly recommend following the update flow as outlined above to avoid any inc
...
-* For direct update to API Portal May 2022 (which is based on Joomla 4), we recommend you to update from the API Portal Feb 2022 (which is based on Joomla 3.10) first.
+* API Portal 7.7.20220830 is compatible with API Gateway and API Manager 7.7.20220830 only.
+* For update to API Portal August 2022 (which is based on Joomla 4), from the API Portal versions older than Feb 2022 we recommend you to upgrade to API Portal Feb 2022 first and then follow upgrade section [Upgrade API Portal May 2022 or newer](/docs/apim_installation/apiportal_install/upgrade_automatic/#upgrade-to-api-portal-may-2022-release).
+* Direct update to API Portal August 2022 can be done only form API Portal May 2022 follwoing the upgrade section [Upgrade from the Joomla! Administrator Interface](/docs/apim_installation/apiportal_install/upgrade_automatic/#upgrade-from-the-joomla-administrator-interface).
+
 * This update is not available as a virtual appliance or as a managed service on Axway Cloud.
 
 ## Important changes
 
-It is important, especially when upgrading from an earlier version, to be aware of the following changes in the behavior or operation of the product in this update.
```
Comment (dginevaAxway) [inline content/en/docs/apim_relnotes/20220830_apip_relnotes.md:50]: @andreamussap I am not sure if this needs to be returned back, i think we have added this to context of new Joomla 4 support , but this is not specific and just a warning. I will leave this to you.

**Comment by jmbermingham** [comment]
Looks good.

File: content/en/docs/apim_relnotes/20220830_apip_relnotes.md (hunk 1 above)
Comment (andreamussap) [reply content/en/docs/apim_relnotes/20220830_apip_relnotes.md:50]: @dginevaAxway The Important changes section is part of the template. If there's no important change in this release, it's ok so say that. But, to confirm with @jmbermingham Thanks

Comment (dginevaAxway) [reply content/en/docs/apim_relnotes/20220830_apip_relnotes.md:50]: @andreamussap  Yeh, right , no important changes for this release.

**Review by jmbermingham — APPROVED** [review]

**Pull request merged**